pkgs/development/compilers/gcc/11/default.nix

```diff
@@ -103,9 +108,15 @@ stdenv.mkDerivation ({
 
   hardeningDisable = [ "format" "pie" ];
 
+  postPatch = ''
+    configureScripts=$(find . -name configure)
+    for configureScript in $configureScripts; do
+      patchShebangs $configureScript
+    done
+  ''
   # This should kill all the stdinc frameworks that gcc and friends like to
```

Comment:
This section moved from pre to post patch, otherwise the changes in here invalidate the patch base.

Reply:
Extracting this into a separate PR targeting staging: #121772.